core/src/main/java/org/apache/iceberg/hadoop/ConfigProperties.java

```diff
@@ -25,4 +25,5 @@ private ConfigProperties() {
   }
 
   public static final String ENGINE_HIVE_ENABLED = "iceberg.engine.hive.enabled";
+
```

Comment:
Nit: this file doesn't need to be touched. Could you remove this whitespace change? It may cause git conflicts.